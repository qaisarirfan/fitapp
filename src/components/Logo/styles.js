import {Dimensions, StyleSheet} from "react-native"

const screen = Dimensions.get("window")

// Logo: Styling here
const styles = StyleSheet.create({
  logoContainer: {
    alignItems: "center",
    justifyContent: "center",
    marginBottom: 40,
  },
  logo: {
    width: screen.width * 0.8,
    height: screen.width * 0.8,
  },
})

export default styles
