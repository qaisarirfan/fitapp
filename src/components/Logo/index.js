import React from "react"
import {Image, View} from "react-native"
import PropTypes from "prop-types"
import styles from "./styles"

import logo from "../../assets/logo.png"

// Logo Component content
export const Logo = () => (
  <View style={styles.logoContainer}>
    <Image source={logo} style={styles.logo} resizeMode="contain" />
  </View>
)

// Logo Proptypes
Logo.propTypes = {}

// Logo Default props
Logo.defaultProps = {}

export default Logo
