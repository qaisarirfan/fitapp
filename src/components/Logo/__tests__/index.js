import React from "react"
import renderer from "react-test-renderer"
import Logo from ".."

const createProps = () => ({
  classes: {},
})

describe("<Logo />", () => {
  it("Does not explode", () => {
    const props = createProps()
    const component = renderer.create(<Logo {...props} />).toJSON()
    expect(component).toMatchSnapshot()
  })
})
