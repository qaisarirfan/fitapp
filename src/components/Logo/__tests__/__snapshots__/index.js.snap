// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> Does not explode 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginBottom": 40,
      },
    ]
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "process": [Function],
      }
    }
    style={
      Object {
        "height": 300,
        "width": 300,
      }
    }
  />
</View>
`;
